src/main/java/org/killbill/billing/invoice/api/InvoiceUserApi.java

```diff
@@ -292,4 +292,13 @@ public InvoiceItem insertInvoiceItemAdjustment(UUID accountId, UUID invoiceId, U
      */
     public void consumeExstingCBAOnAccountWithUnpaidInvoices(final UUID accountId, final CallContext context);
 
+    /**
+     * Move the invoice status form DRAFT to COMMITTED
+     *
+     * @param invoiceId invoive id
+     * @param context the tenant context
+     * @throws InvoiceApiException
+     */
+    public void invoiceStatusTransition(UUID accountId, UUID invoiceId, CallContext context) throws InvoiceApiException;
```

Comment:
* Do we need the `accountId` parameter? Isn't `invoiceId` enough?
* Regarding naming, let's find something more explicit. Maybe `commitInvoice`?